### ISSUE TITLE
No native build was found?

### ISSUE THREAD

**Issue by Alecyrus** [emilbayes/macos-touchid#1]
<img width="702" alt="截屏2019-09-2009 08 06" src="https://user-images.githubusercontent.com/13865581/65291486-2be46600-db86-11e9-9afe-d8924ac739f4.png">

<img width="929" alt="截屏2019-09-2009 06 31" src="https://user-images.githubusercontent.com/13865581/65291435-f2135f80-db85-11e9-9431-205e8deab540.png">

**Comment by emilbayes** [comment]
Just tried this on latest electron. Maybe electron-builder is messing with the prebuilds?

![image](https://user-images.githubusercontent.com/416524/65601719-e3241700-dfa2-11e9-8f7f-940fe7c073ea.png)

**Comment by Alecyrus** [comment]
Still the previous error.  I guess that's because my macOS version is not stable version?

**Comment by kgroat** [comment]
It seems like you are trying to use this module from the [render process and not the main process](https://www.electronjs.org/docs/tutorial/application-architecture).  For non modules with compiled native bindings, such as this one, you can *only* use them in the main process.

**Comment by emilbayes** [comment]
@kgroat That's not correct. The prebuilds work both in the main and the render process. But they might be outdated for latest version of electron now
